libraries/AP_BattMonitor/AP_BattMonitor_Params.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <AP_Param/AP_Param.h>
+
+class AP_BattMonitor_Params {
+public:
+    static const struct AP_Param::GroupInfo var_info[];
+
+    AP_BattMonitor_Params(void);
+
+    constexpr AP_BattMonitor_Params(AP_BattMonitor_Params &&other) = default;
```

Comment:
I think it doesn't hurt, but this has also been removed from the other classes when the create was removed.

Reply:
Addressed.